argocd_observability/deployment_event.md

```diff
@@ -0,0 +1,71 @@
+### ArgoCD Sync Deployment event
+
+| Property | Source | in Dashboard used | 
+|---|---|---|
+| event.id  |   |
+| event.kind = "SDLC_EVENT" |   | x 
+| event.version = "0.1.0" | |  x
+| event.category = "task" | |  x
+| event.type = "deployment" | |  x
+| event.provider = "argocd" | |  x
+| event.status = started/finished | |  x
+| duration | |  x
+| start_time | |  x
+| end_time | |  x
+| timestamp | |  x
+| | | |
+| task.id | | -
+| task.name | | -
+| task.outcome | must be mapped from argocd.sync.status | x
```

Comment:
I think sync status is the wrong value here, as it is one of
* synced
* out of sync
* unknown
@johannes-b can you pls clarify on that?

Reply:
Correct. For the use case of ArgoCD sending the deployment event, it would make more sense using the `app.status.operationState.phase` property.